### PR TITLE
RotoTrans Savior

### PR DESCRIPTION
**Pull request by pariterre**
There was a mistake in XYZ sequence fromEuler. It is now solved!

### CODE REVIEW

File: pyomeca/frame_dependent.py [hunk 1]
```diff
@@ -494,7 +494,9 @@ def _parse_c3d(c3d, prefix):
         raise NotImplementedError("_parse_c3d_info is an abstract function")
 
     @classmethod
-    def from_c3d(cls, filename, idx=None, names=None, prefix=None):
+    def from_c3d(
```
Comment (romainmartinez) [inline pyomeca/frame_dependent.py]: I think you included an old PR that we refused

Comment (pariterre) [reply pyomeca/frame_dependent.py]: This is a monster that will always strike back!

File: pyomeca/rototrans.py [hunk 2]
```diff
@@ -102,25 +102,29 @@ def get_euler_angles(self, angle_sequence):
...
         elif angle_sequence == "yzx":
             angles[2, :, :] = np.arctan2(-self[1, 2, :], self[1, 1, :])
             angles[0, :, :] = np.arctan2(-self[2, 0, :], self[0, 0, :])
-            angles[1, :, :] = np.arcsin(self[1, 2, :])
+            angles[1, :, :] = np.arcsin(self[1, 0, :])
         elif angle_sequence == "zxy":
             angles[1, :, :] = np.arcsin(self[2, 1, :])
             angles[2, :, :] = np.arctan2(-self[2, 0, :], self[2, 2, :])
             angles[0, :, :] = np.arctan2(-self[0, 1, :], self[1, 1, :])
+        elif angle_sequence == "zyx":
```
[the remark this answers is not in the document]
Comment (pariterre) [reply pyomeca/rototrans.py]: no it is not! Let me fix that!

[the remark this answers is not in the document]
Comment (pariterre) [reply pyomeca/rototrans.py]: you actually looked at an old commit it seems... the last commit should include the yxz sequence

**Comment by romainmartinez** [comment]
Ok you can merge if you're done :+1:

**Pull request merged**